src/UiPath.Workflow.Runtime/ActivityUtilities.cs

```diff
@@ -1184,18 +1184,18 @@ public class ActivityCallStack
         private int _nonExecutingParentCount;
         private readonly Quack<ChildActivity> _callStack;
 
-        public ActivityCallStack()
```

Comment:
change ActivityCallStack to internal and all items in it can remain unchanged

Reply:
Explained in `ChildActivity` comment the situation.